File: Python/loops.py
for i in range(1, 5):
    print("for loop iteration", i)

num_list = range(1, 5)
for i in num_list:
    print("for loop with list iteration", i)

i = 1
while i < 5:
    print("while loop iteration", i)
    i += 1

# "Do while" for Python
i = 1
while True:
    print("do while loop iteration", i)
    i += 1
    if i > 4:
        break

File: C/loops.c
#include <stdio.h>

int main()
{

   for (int i = 1; i < 5; i++)
   {
      printf("for loop iteration %d\n", i);
   }

   int i = 1;
   while (i < 5)
   {
      printf("while loop iteration %d\n", i);
      i++;
   }

   i = 1;
   do
   {
      printf("do while loop iteration %d\n", i);
      i++;
   } while (i < 5);

   return 0;
}